### PR TITLE
Gitignore

### PR DESCRIPTION
**Pull request by hridhi**
## Related Issue 

- https://github.com/shagun25/SheHeroes/issues/58

Closes: #58

### Describe the changes you've made

Added few lines from the reference 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.

## Additional Info (optional)
Part of GSSoC'21

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,29 +17,95 @@
...
+**/android/key.properties
+*.jks
+
+# iOS/XCode related
+**/ios/**/*.mode1v3
+**/ios/**/*.mode2v3
+**/ios/**/*.moved-aside
+**/ios/**/*.pbxuser
+**/ios/**/*.perspectivev3
+**/ios/**/*sync/
```
Comment (hiashutoshsingh) [inline .gitignore:69]: add pod lock file for ios also here

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Looks good.

Just a small change needed.

File: .gitignore (hunk 1 above)
Comment (hridhi) [reply .gitignore:69]: **/ios/**/Pods/ 
this one in line 75 is for pod lock ig

Comment (hiashutoshsingh) [reply .gitignore:69]: yes this one also, and in the root folder of ios there is a Podfile.lock file 

Screenshot 2021-03-30 at 10 13 34 AM

Comment (hiashutoshsingh) [reply .gitignore:69]: hey @hridhi any update on the above, its just a small change and then we are good for merge.

Comment (hridhi) [reply .gitignore:69]: sorry for delay please check

**Review by hiashutoshsingh — APPROVED** [review]
Thanks for the contribution

**Pull request merged**